src/main/java/nl/tudelft/dnainator/ui/widgets/AboutDialog.java

```diff
@@ -0,0 +1,83 @@
+package nl.tudelft.dnainator.ui.widgets;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import javafx.scene.control.ButtonBar.ButtonData;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Stage;
+
+/**
+ * Creates a {@link Dialog} displaying information about the
+ * application.
+ */
+public class AboutDialog {
+	private Dialog<String> dialog;
+	private Properties prop;
+	private static final String ICON = "/ui/icons/dnainator128x128.png";
+	private static final String STYLE = "/ui/style.css";
+	private static final String PROPERTIES = "config.properties";
+	
+	/**
+	 * Instantiates a new AboutDialog.
+	 * It will set up all the necessities for displaying 
+	 * information of the application.
+	 */
+	public AboutDialog() {
+		setupDialog();
+	}
+
+	private void setupDialog() {
+		dialog = new Dialog<String>();
+		dialog.setTitle("About DNAinator");
+		dialog.getDialogPane().getStylesheets().add(getClass().getResource(STYLE).toString());
+		Stage stage = (Stage) dialog.getDialogPane().getScene().getWindow();
+		Image icon = new Image(this.getClass().getResourceAsStream(ICON));
```

Comment:
2 lines up you don't use `this`, here you do. Try to be consistent in your style.